migrations/versions/10_adding_supplier_to_user_table.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: 10_adding_supplier_to_user_table
```

Comment:
I'm tempted to get https://github.com/alphagov/digitalmarketplace-api/pull/77 merged before this one (updating the suppliers table for supplier import) - it also contains a migration so the down-revision here would need to change.